src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/MockGoogleDataprocDAO.scala

```diff
@@ -27,4 +35,34 @@ class MockGoogleDataprocDAO extends DataprocDAO {
       clusters.remove(clusterName)
```

Comment:
nit: `remove` doesn't barf if the key doesn't exist, so you don't need the if here